src/app/modules/startup/controller.nim

```diff
@@ -170,6 +170,10 @@ proc init*(self: Controller) =
     self.delegate.onLocalPairingStatusUpdate(args)
   self.connectionIds.add(handlerId)
 
+  handlerId = self.events.onWithUUID(SIGNAL_REENCRYPTION_PROCESS_STARTED) do(e: Args):
+    self.delegate.onReencryptionProcessStarted()
+  self.connectionIds.add handlerId
```

Comment:
Looks like a syntax error to me?

Reply:
the funny think - I don't have any errors

Reply:
fixed

Reply:
Yeah... I'm always impressed how NIM sometimes compiles code with syntax errors without making a sweat

Reply:
This turns out not to be a syntax error in Nim at all: https://play.nim-lang.org/#ix=4s9e

Reply:
Nim does have a lot of "freedom" in their syntax haha.
those all work:
```nim
things.add(stuff)
add(things, stuff)
things.add stuff
```

Reply:
nice... :D